atelier-core/src/builder/mod.rs

```diff
@@ -242,6 +245,18 @@ impl ModelBuilder {
             {
                 self.prelude_namespace.make_shape(shape_name)
             } else {
+                // we couldn't find this shape name
+                let mut known_shape_names = prelude::prelude_model_shape_ids(&self.smithy_version)
+                    .iter()
+                    .map(|s| format!("{}", s))
+                    .collect::<Vec<String>>();
+                known_shape_names.sort();
+                debug!(
```

Comment:
wouldn't this be error? OR have a smaller error message and the longer debug to follow.